common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4564,7 +4564,7 @@ public static enum ConfVars {
     HIVE_DECODE_PARTITION_NAME("hive.decode.partition.name", false,
         "Whether to show the unquoted partition names in query results."),
 
-    HIVE_EXECUTION_ENGINE("hive.execution.engine", "mr", new StringSet(true, "mr", "tez"),
+    HIVE_EXECUTION_ENGINE("hive.execution.engine", "tez", new StringSet(true, "tez", "mr"),
         "Chooses execution engine. Options are: mr (Map reduce, default), tez. While MR\n" +
```

Comment:
Can we update the text as well,
"Chooses execution engine. Options are: tez (default), mr(Map reduce). MR\n" +
        "is a historical engine\n" +
        "and is deprecated in Hive 2 line. It may be removed without further warning."),

Reply:
sure, this first commit is just for a quick precommit sanity test

Reply:
Make sense, thx

Reply:
this has been addressed in recent commits